app/containers/SignIn/index.js

```diff
@@ -1,5 +1,5 @@
 // @flow
-import React, { Component } from 'react';
+import React, { Component } from "react";
```

Comment:
```suggestion
import React, { Component } from 'react';
```

Please use single quotes in all the places where you have used double quotes, since the coding styles should be consistent across the project.